src/algorithms/textrank.py

```diff
@@ -2,13 +2,10 @@
 
 from gensim.summarization import summarizer
 
+
 class TextRank():
-    def __init__(self, text, ratio=0.2, word_count=None, split=False):
+    def __init__(self, text):
         self.text = text
-        self.ratio = ratio
-        self.word_count = word_count
-        self.split = split
```

Comment:
The parameters have to be taken from the UI, so the constructor has to provide an option for them.

Reply:
Yeah, I removed the parameters by mistake. I've added them back.